stream-chat-android-client/src/main/java/io/getstream/chat/android/client/extensions/StringExtensions.kt

```diff
@@ -25,3 +25,16 @@ internal fun String.camelCaseToSnakeCase(): String {
 }
 
 internal fun String.isAnonymousChannelId(): Boolean = contains("!members")
+
+/**
+ * Parses CID of channel to channelType and channelId.
+ *
+ * @return Pair<String, String> Pair with channelType and channelId.
+ * @throws IllegalStateException Throws an exception if format of cid is incorrect.
+ */
+@Throws(IllegalStateException::class)
+public fun String.cidToTypeAndId(): Pair<String, String> {
+    require(isNotEmpty()) { "cid can not be empty" }
```

Comment:
`require` throws `IleegalArgumentException` so I think we should either change all `require` to `check` (which throws `IllegalStateException`) or change thrown exception type

Reply:
Nice catch! Thanks! Fixed it